protocol/contracts/interfaces/oud/Oud__ITempleLineOfCredit.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.8.17;
```

Comment:
Why do these interfaces have the `Oud__` prefix? I think can just make a:

`contracts/interfaces/templeLineOfCredit/ITempleLineOfCredit.sol`

as those interfaces may be of use to other things in the repo later

Reply:
Was following the Ramos/AMO pattern

Reply:
fixed in [8f466df](https://github.com/TempleDAO/temple/pull/760/commits/8f466dfaf4fcc70fc1f29c24fb79fb2ddf9ef9dc)?

Reply:
fixed in [8f466df](https://github.com/TempleDAO/temple/pull/760/commits/8f466dfaf4fcc70fc1f29c24fb79fb2ddf9ef9dc)